docs/mindsdb_sql/knowledge-bases.mdx

```diff
@@ -116,6 +125,10 @@ The following is an explanation of the syntax and parameters:
 * `<id_col>` is a column that contains unique identifiers of data. The knowledge base expects the `id` column name, unless specified differently in the `id_column` parameter when creating the knowledge base.
 * `<text_col>` is a column that contains the text content. The knowledge base expects the `content` column name, unless specified differently in the `content_columns` parameter when creating the knowledge base.
 
+<Note>
+Note that when inserting the same data into the knowledge base again, then no new rows are inserted, as the content already exists in the knowledge base.
```

Comment:
when inserted row has existed id but different content - the existed record will be updated (embeddings and content)

Reply:
Added this:

In order to update existing data in the knowledge base, insert data with the column ID that you want to update and the updated content.